src/Microsoft.ML.Data/Transforms/TransformsCatalog.cs

```diff
@@ -12,10 +12,29 @@ public sealed class TransformsCatalog
     {
         internal IHostEnvironment Environment { get; }
 
+        /// <summary>
+        /// The list of operations over categorical data.
+        /// </summary>
         public CategoricalTransforms Categorical { get; }
+
+        /// <summary>
+        /// The list of operations for data type conversion.
+        /// </summary>
         public ConversionTransforms Conversion { get; }
+
+        /// <summary>
+        /// The list of operations for processing text data.
+        /// </summary>
         public TextTransforms Text { get; }
+
+        /// <summary>
+        /// The list of operations for data projection.
+        /// </summary>
         public ProjectionTransforms Projection { get; }
+
+        /// <summary>
+        /// The list of operations for selecting features based on some criteria.
```

Comment:
>based on some criteria [](start = 58, length = 22)

I feel its unnecessary here because for rest of the properties we don't say any details.